ui/app/templates/vault/cluster.hbs

```diff
@@ -79,8 +79,8 @@
         <div class="navbar-sections">
           <div class={{if (is-active-route "vault.cluster.clients") "is-active"}}>
             <LinkTo
-              @route="vault.cluster.clients.history"
-              @current-when="vault.cluster.clients.history"
+              @route="vault.cluster.clients.dashboard"
+              @current-when="vault.cluster.clients.dashboard"
```

Comment:
I'm curious if this is needed? The component should consider the link active automatically when the url matches the route and `current-when` is typically used to make the link active when it doesn't match.

Reply:
I think you're right. I'll make a ticket to cleanup all of the `@current-when` args, since the other `<LinkTo>` components in this file, and a few elsewhere, still use them.